src/main/java/com/auth0/json/mgmt/authenticationmethods/AuthenticationMethod.java

```diff
@@ -0,0 +1,247 @@
+package com.auth0.json.mgmt.authenticationmethods;
```

Comment:
I think these classes should go in the `json.mgmt.users` package, unless used outside of the UsersEntity.